.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+
+<!-- Please describe the issue in detail here, and for bug reports fill in the
+fields below -->
```

Comment:
Something indicating that the entire template can be deleted for a feature request would be nice.

Reply:
added